### PR TITLE
Tobias Hagberg, Team Tigers, Class Jan21

### PR DESCRIPTION
**Pull request by TobiasHagberg**
Deployed project: https://th-project-mongodb.herokuapp.com/

EXAMPLES

ALL SALES: /sales
QUERY: sales?region=BaltimoreWashington
SINGLE ENTRY: sales/_id

- If I had more time
With more time I would have liked looking into how to specify a query or path or perhaps a frontend solotion to:
 - Average price over time
 - Top ten total volume
 - highest notation (combination of avg price and volume)
 - graphical representation from frontend

- Difficulties
The set up of accounts on MongoDB, Compas, Atlas and Heroku was the big hick-up this week. Other than that the code sessions included more than enough guidance and pointers to bring this project home :)

### CODE REVIEW

**Review by Yennie-Leek — APPROVED** [review]
Good job on the project! 🥳
Looks nice, clean and structured  🙌
Works well too 😀

File: README.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+QUERY: sales?region=BaltimoreWashington
+SINGLE ENTRY: sales/_id
+
+
+- If I had more time
+With more time I would have liked looking into how to specify a query or path or perhaps a frontend solotion to:
+ - Average price over time
+ - Top ten total volume
+ - highest notation (combination of avg price and volume)
+ - graphical representation
```
Comment (KarolCislowski) [inline README.md:32]: nice short documentation 🥳

File: server.js [hunk 2]
```diff
@@ -1,38 +1,88 @@
...
+
+app.get('/sales', async (req, res) => {
+  const { region } = req.query
+
+  if (region) {
+    const avocadoSales = await AvocadoSale.find({
+      region: {
+        $regex: new RegExp(region, "i")
+      }
+    })
```
Comment (KarolCislowski) [inline server.js:72]: nice sorting 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 🥇

**Pull request closed without merging**